components/user-header.js

```diff
@@ -207,7 +207,7 @@ function HeaderNym ({ user, isMe }) {
     : <NymView user={user} isMe={isMe} setEditting={setEditting} />
 }
 
-function SocialLink ({ name, id }) {
+export function SocialLink ({ name, id }) {
```

Comment:
`export` no longer needed

Reply:
fixed

Fix will be available on my next push